library/data/component/src/main/resources/quilt_component.mixins.json

```diff
@@ -0,0 +1,25 @@
+{
+  "required": true,
+  "package": "org.quiltmc.qsl.component.mixin",
+  "compatibilityLevel": "JAVA_17",
+  "mixins": [
+    "RegistryAccessor",
```

Comment:
This errors? Does this mixin even exist?

Reply:
Woops my bad!